tests/integration/targets/virt/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- include_vars: '{{ item }}'
+  with_first_found:
+    - "{{ ansible_distribution }}-{{ ansible_distribution_version}}.yml"
+    - "{{ ansible_distribution }}-{{ ansible_distribution_major_version}}.yml"
+    - "{{ ansible_distribution }}.yml"
+    - "default.yml"
```

Comment:
could we also add `{{ ansible_os_family }}` and `{{ ansible_os_family }}-{{ ansible_distribution_major_version }}` checks here before `default` here? It might make it easier to test things like Rocky without having to duplicate the yml files unnecessarily.